### DISCUSSION TITLE
cant find the code in the repo

### DISCUSSION THREAD

**Discussion by Lucian-Ryu** [Cyfrin/foundry-full-course-cu#241] [General]
hello everyone, im having an issue sice couple of days were i cant find the code used in the video in the github repo. could anyone point me to where the code is at? 

thank you very much

**Comment by shikhar229169** [comment]
Go to this link: https://github.com/Cyfrin/foundry-full-course-f23#table-of-contents
Over there click the lesson for which you want to browse the code.
And click on the 'code' link.

**Reply by Lucian-Ryu** [reply]
thank you very much!
